docs/features/commands.rst

```diff
@@ -3,45 +3,196 @@ Commands
 
 .. _cmd: https://docs.python.org/3/library/cmd.html
 
-How to create a command with a ``do_command`` method,
+``cmd2`` is designed to make it easy for you to create new commands. These
+commmands form the backbone of your application. If you started writing your
+application using cmd_, all the commands you have built will work when you move
+to ``cmd2``. However, there are many more capabilities available in ``cmd2``
+which you can take advantage of to add more robust features to your commands,
+and which makes your commands easier to write. Before we get to all the good
+stuff, let's briefly discuss how to create a new command in your application.
 
-Parsed statements
------------------
 
-``cmd2`` passes ``arg`` to a ``do_`` method (or ``default``) as a Statement, a
-subclass of string that includes many attributes of the parsed input:
+Basic Commands
```

Comment:
@kotfu You did a great job improving the documentation in this file.  I really like how it flows.